### PR TITLE
MBO - PS9 compatibility

### PR DESCRIPTION
**Pull request by sowbiba**

### CODE REVIEW

**Comment by matks** [comment]
@intraordinaire What will happen to this PR 😄 ? I can't wait

**Comment by matks** [comment]
👀

**Comment by intraordinaire** [comment]
@matks 
> 👀

In progress, but we still have some small issues on our side.

**Comment by matks** [comment]
Don't hesitate to share the issues, we can try to help 😉 

Us or ChatGPT 😆

File: ps_mbo.php [hunk 1]
```diff
@@ -101,7 +102,7 @@ public function __construct()
         $this->need_instance = 0;
         $this->ps_versions_compliancy = [
             'min' => '8.0.2',
```
Comment (intraordinaire) [inline ps_mbo.php:104]: Need to update the min also no ?

File: src/Traits/Hooks/UseDashboardZoneOne.php [hunk 2]
```diff
@@ -130,8 +133,11 @@ protected function loadPsAccounts(): string
      * 
      * @return bool
      */
-    private function ensurePsAccountIsEnabled(): bool {
-        $accountsInstaller = $this->get('mbo.ps_accounts.installer');
+    private function ensurePsAccountIsEnabled(): bool
+    {
+        if (version_compare(_PS_VERSION_, "9.0.0", ">=")) return false;
```
Comment (intraordinaire) [inline src/Traits/Hooks/UseDashboardZoneOne.php:138]: Don't forget to remove this after publication of v9.0.0 on addons and/or official releases for the module

File: src/Traits/Hooks/UseDashboardZoneOne.php [hunk 3]
```diff
@@ -143,6 +149,8 @@ private function ensurePsAccountIsEnabled(): bool {
 
     private function ensurePsEventbusEnabled()
     {
+        if (version_compare(_PS_VERSION_, "9.0.0", ">=")) return false;
```
Comment (intraordinaire) [inline src/Traits/Hooks/UseDashboardZoneOne.php:152]: Don't forget to remove this after publication of v9.0.0 on addons and/or official releases for the module

**Pull request closed without merging**